lib/utils.js

```diff
@@ -1,4 +1,4 @@
-import { isFunction, template, union } from "lodash-es";
+import { template, union } from "lodash-es";
```

Comment:
`isFunction` was imported but not used.

Reply:
Incase the unused function is desired
```suggestion
import { isFunction, template, union } from "lodash-es";
```